composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "library",
     "description": "Bolt 4 skeleton / prototype",
     "require": {
-        "php": ">=7.1.3",
+        "php": ">=7.2",
```

Comment:
SF 4 uses 7.1.3. We should stick with that. https://symfony.com/doc/current/reference/requirements.html

Reply:
PHPUnit 8 requires PHP 7.2 and 7.1 is close to EOL. But I can revert it, no problem.

Reply:
reverted

Reply:
Thanks. I think it's beneficial to us to stick with SF's version requirements :-)